vue/components/ui/atoms/button-color/button-color.vue

```diff
@@ -355,10 +357,6 @@ export const ButtonColor = {
             type: Boolean,
             default: false
         },
-        loaderStyle: {
-            type: Object,
-            default: () => ({})
-        },

```

Comment:
This prop shouldn't probably exist in the first place, but as it's there, I don't think we should remove it now to ensure we are maintaining retrocompatibility.

Just do this in the newly created computed variable `loaderStyle` (you will need to change it's name too):
```
        _loaderStyle() {
            const base = {};
            base["background-color"] = this.color === "white" ? "#2d2d2d" : "#e4e8f0";
            base = { ...base, ...this.loaderstyle };
            return base;
        }
```